exporter/awscloudwatchlogsexporter/README.md

```diff
@@ -44,21 +42,23 @@ exporters:
     log_stream_name: "testing-integrations-stream"
 ```
 
-All configuration options:
+Example configuration for EMF logs:
 
 ```yaml
 exporters:
   awscloudwatchlogs:
-    log_group_name: "testing-logs"
-    log_stream_name: "testing-integrations-stream"
+    log_group_name: "testing-logs-emf"
+    log_stream_name: "testing-integrations-stream-emf"
+    raw_log: true
     region: "us-east-1"
-    role_arn: "arn:aws:iam::123456789:role/monitoring-application-logs"
     endpoint: "logs.us-east-1.amazonaws.com"
     log_retention: 365
     tags: { 'sampleKey': 'sampleValue'}
-    sending_queue:
-      queue_size: 50
-    retry_on_failure:
-      enabled: true
-      initial_interval: 10ms
 ```
+
+## Additional Notes 
+
+- If the log group and/or log stream are specified in an EMF log, that EMF log will be exported to that log group and/or log stream (i.e. ignores the log group and log stream defined in the configuration)
+  - The log group and log stream will also be created automatically if they do not already exist
+- [ARNs](https://docs.aws.amazon.com/IAM/latest/UserGuide/reference-arns.html) are currently not supported with the CloudWatch Logs Exporter
+- An error is raised if the log group and log stream specified (either in the configuration or an EMF log) already exist, but only an error log message is produced. Execution of the CloudWatch Logs Exporter doesn't stop when this error is raised.
```

Comment:
can we say that a warning message is produced if the log group/stream already exists?  Is it a error log or a warning log? Can we move from error to warning?

Reply:
Actually, I believe it's a debug log. For context this is the message that gets produced:
`2023-06-28T22:47:32.304Z        debug   cwlogs@v0.79.0/cwlog_client.go:145      cwlog_client: creating stream fail      {"kind": "exporter", "data_type": "logs", "name": "awscloudwatchlogs", "error": "ResourceAlreadyExistsException: The specified log stream already exists"}`